.github/workflows/build.yml

```diff
@@ -49,6 +49,15 @@ jobs:
             skip-crashpad: false
             build-appimage: true
             build-deb: true
+          - os: ubuntu-24.04
+            container: ghcr.io/chatterino/chatterino2-build-ubuntu-24.04:latest
+            qt-version: 6.6.1
+            force-lto: false
+            plugins: true
+            skip-artifact: false
+            skip-crashpad: false
+            build-appimage: false
```

Comment:
I don't know how AppImages work exactly, but couldn't we build the AppImage on 24.04 now?

Reply:
I don't know either, but I disabled it because I had conflicts with the filename in CI and iirc pajlada said something on stream about disabling it.

Reply:
The reason I don't want to "just" bump the AppImage is that generally, the AppImage is built on older versions since that ensures it runs on older distros due to some glibc magic. We build it in 22.04 instead of 20.04 because of 22.04 using qt6 before and 20.04 used qt5, and we didn't want to distribute qt5 there